ppcdis/elf2rel.py

```diff
@@ -111,6 +111,8 @@ def _get_sections_to_link(self) -> List[int]:
             if sec["sh_type"] in ("SHT_PROGBITS", "SHT_NOBITS")
             and sec["sh_flags"] & SH_FLAGS.SHF_ALLOC
             and sec.name not in  ("forcestrip", "relsymdef")
+            # TODO: link sections by name instead of index
+            and sec.name in (".text", ".ctors", ".dtors", ".data", ".rodata", ".bss")
```

Comment:
What's the point of this?

Reply:
Prevents sections other that those from being created. mkw decomp has a helper "binary_blobs" section that looks like a data section so `elf2rel.py` emits it. But yeah it's pretty bad code if the REL format can contain other sections.

Reply:
I think a better solution might be having a command line flag where you can specify extra section names to skip, I've not saw a real rel use a custom section yet but they definitely could. I don't mind just adding that myself after merging if you'd like. Not sure what you mean by the todo btw?

Reply:
agreed, I will commit some time today. The todo is from when I didn't know to how to force section order with -r1, I'll delete it.